src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx

```diff
@@ -72,6 +72,20 @@ export default function QueryEditorTopRow(props: QueryEditorTopRowProps) {
     },
   } = opensearchDashboards.services;
 
+  useEffect(() => {
+    function handleCmdEnter(event: KeyboardEvent) {
```

Comment:
I think we could use 

this.inputRef which is the Monaco Editor ref and add an action like [this](https://microsoft.github.io/monaco-editor/playground.html?source=v0.52.0#example-interacting-with-the-editor-adding-an-action-to-an-editor-instance)

Reply:
Have updated it